index.html

```diff
@@ -46,181 +56,249 @@ <h2 class="sub-title">I'm a Software Developer</h2>
+                        <img class="image-bg-mobile" src="assets/about/bg_img.png" alt="bg-img"/>               
                     </article>
-                    <article>
+                    <article class="skills">
                         <hr class="about-hr">
                         <ul class="my-stack">
                             <span class="stack-title">
                                 <img src="assets/about/Rectangle_55.png" alt="shape">
                                 <h3>Language</h3>
+                                <div class="lang-frameworks1">
+                                    <li class="stack3">JavaScript</li>
+                                    <li class="stack3">Ruby</li>
+                                    <li class="stack3">HTML</li>
+                                    <li class="stack3">CSS</li>
+                                </div>
                             </span>
-                            <ul class="lang-frameworks">
-                                <li class="stack3">JavaScript</li>
-                                <li class="stack3">Ruby</li>
-                                <li class="stack3">HTML</li>
-                                <li class="stack3">CSS</li>
-                            </ul>
                             <span class="stack-title">
                                 <img src="assets/about/Rectangle_56.png" alt="shape">
                                 <h3>Frameworks</h3>
+                                <div class="lang-frameworks2">
+                                    <li class="stack3">React.js</li>
+                                    <li class="stack3">Ruby on Rails</li>
+                                    <li class="stack3">RSpec</li>
+                                    <li class="stack3">Capybara</li>
+                                </div>
                             </span>
-                            <ul class="lang-frameworks">
-                                <li class="stack3">React.js</li>
-                                <li class="stack3">Ruby on Rails</li>
-                                <li class="stack3">RSpec</li>
-                                <li class="stack3">Capybara</li>
-                            </ul>
                             <span class="stack-title">
                                 <img src="assets/about/Rectangle_57.png" alt="shape">
                                 <h3>Skills</h3>
+                                <div class="lang-frameworks3">
+                                    <li class="stack3">Database Management</li>
+                                    <li class="stack3">Version Control</li>
+                                    <li class="stack3">CLI</li>
+                                    <li class="stack3">API Design</li>
+                                    <li class="stack3">Web Development</li>
+                                </div>
```

Comment:
Kindly note that their seems to be an issue with this section as the technologies section is missing which is close to the contact-me section. Please check your design and fix it.

Your design :
![image](https://github.com/allenkays/Portfolio-setup-and-mobile-first/assets/20537712/9caa869e-8713-446b-a53b-9417cbc41518)

Figma design :

![image](https://github.com/allenkays/Portfolio-setup-and-mobile-first/assets/20537712/58e6d193-65a7-4bf3-89fa-99794e2ec871)